sheer/query.py

```diff
@@ -304,8 +306,23 @@ def get_document(doctype, docid):
         raw_results = es.get(index=es_index, doc_type=doctype, id=docid)
         return QueryHit(raw_results)
 
+    def convert_to_datetime(timestamp):
+        date = parser.parse(timestamp)
+        return date.astimezone(timezone('UTC'))
+
+    def when(starttime, endtime):
+        start = convert_to_datetime(starttime['date'])
```

Comment:
(just inferring from the code), it should take any string that [dateutil.parser](https://labix.org/python-dateutil#head-c0e81a473b647dfa787dc11e8c69557ec2c3ecd2) can deal with